ansible/examples/slave.yml

```diff
@@ -627,6 +631,29 @@
             line: '%dist .el{{ ansible_distribution_major_version }}'
       when: ansible_os_family == "RedHat"
 
+    ## tmpwatch/tmpreaper TASKS
+    - name: tmpwatch/tmpreaper Tasks
+      block:
+        # In case we're running 'ansible-playbook --tags tmp'
+        - name: "Make sure {{ tmp_cleaner_name }} is installed"
+          package:
+            name: "{{ tmp_cleaner_name }}"
+            state: present
+
+        - name: Disable tmpreaper cron.daily timer
+          file:
+            path: /etc/cron.daily/tmpreaper
+            state: absent
+          when: ansible_os_family == "Debian"
+
+        - name: Create tmp cleaning cronjob
+          cron:
+            name: "Clean /tmp"
+            special_time: daily
+            job: "{{ tmp_cleaner_name }} 30d /tmp/"
```

Comment:
Does this mean run daily and remove anything that's 30 days old or older?  If so a comment wouldn't be out of place for obviousness without having to look up the utility

Reply:
Fair enough